mapdrawgdal.c

```diff
@@ -1998,6 +2001,13 @@ msDrawRasterLayerGDAL_16BitClassification(
     }
   }
 
+  /* -------------------------------------------------------------------- */
+  /*      Fetch the scale classification option.                          */
+  /* -------------------------------------------------------------------- */
+  pszClassifyScaled = CSLFetchNameValue( layer->processing, "CLASSIFY_SCALED" );
+  if( pszClassifyScaled != NULL && EQUAL(pszClassifyScaled,"TRUE") )
```

Comment:
Perhaps use
```
if( pszClassifyScaled != NULL )
   bClassifyScale = CSLTestBoolean(pszClassifyScaled);
```
so as to accept TRUE, YES, ON, 1

Reply:
Ok, cool - a good pattern to know about.